app/src/features/apiClient/helpers/auth.ts

```diff
@@ -4,35 +4,38 @@ import { KeyValuePair, RQAPI } from "../types";
 import { Authorization } from "../screens/apiClient/components/clientView/components/request/components/AuthorizationView/types/AuthConfig";
 import { getDefaultAuth } from "../screens/apiClient/components/clientView/components/request/components/AuthorizationView/defaults";
 
-export const processAuthForEntry = (
+export const getEffectiveAuthForEntry = (
   entry: RQAPI.Entry,
   entryDetails: {
     id: RQAPI.Record["id"];
     parentId: RQAPI.Record["collectionId"];
   },
   allRecords: RQAPI.Record[]
 ) => {
-  const entryCopy = JSON.parse(JSON.stringify(entry)) as RQAPI.Entry; // Deep Copy
-
+  // this mainly finds out the auth type in case of "INHERIT"
```

Comment:
forgot to mention the detail in [the comment](https://github.com/requestly/requestly/pull/2966/files#r2079029010) that the doc string was meant to justify passing `allRecords` as an argument